runtimerevolution/web3-roulette
# [WEB3-9] Admin Giveaways Empty State

# Description
This PR adds the empty state page for admin users.

It also includes:
- update create new button;
- adjust giveaways list page responsiveness.

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-9

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Test case 1
1. Remove all the giveaways from the database
2. Set the role of the user as admin
3. Login and verify the landing page
4. Verify if the layout correspond to the design

Other scenarios:
- verify the layout in mobile view
- verify the landing page as a normal user, will be in another PR
- verify the updated create new button in the giveaways list page

## Checklist:
